plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestTimestampPrecision.java

```diff
@@ -52,16 +52,16 @@
 import static org.apache.hadoop.hive.serde.serdeConstants.SERIALIZATION_LIB;
 import static org.assertj.core.api.Assertions.assertThat;
 
```

Comment:
It looks like we don't have a way to write this data via trino-parquet, we should add support for that.
There should be a test in AbstractTestParquetReader similar to testTimestamp but with time zone. It'll need update to `ParquetTester#writeValue` to add support for timestamp with timezone type.
There should also be a test in TestHiveCompatibility once the write path is working.

Reply:
Writing TIMESTAMP WITH TIME ZONE isn't  supported by Hive connector anyway yet - I'm planning to implement this in the next PR.